angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead.stories.ts

```diff
@@ -48,35 +65,260 @@ is 54rem (864px). If the content needs of your application
 require this breakpoint to be larger or smaller, it can be
 changed in your application’s Sass file with the
 variable \`$sprk-masthead-breakpoint\`.
-- In Default Masthead the little nav is the section to the
+- In the Default Masthead the Navigation Items section is to the
 right of the logo, containing the navigation links and utility
 items. When the number of navigation items is too large to
-fit in one row, the Extended Masthead utilizes a second row,
-the big nav, to display the navigation items.
-- The Masthead component has 4 slots to inject markup
-into the component template.
-    - \`little-nav-slot\`: Used for adding navigation
-    to the Default Masthead.
-    - \`utility-slot\`: Used to add optional features
-    like account menu, sign in button, search, etc.
-    - \`logo-slot\`: Used for adding your logo.
-    - \`navItem-slot\`: Used for the link in the upper right
-    of the mobile version of the masthead (typically a sign in link)
+fit in one row, the Extended Masthead should be used as it utilizes a second row
+to display the navigation items.
 - The Masthead is a navigation landmark for accessibility
-tools. The attribute \`role=”banner”\` must be present.
+tools and we add the attribute \`role=”banner”\` to the Masthead.
 - The Masthead will be "sticky" on small viewports in all browsers
 except for IE11.
 `,
     docs: { iframeHeight: 300 },
   },
 };
 
+const logoSvg = `
```

Comment:
I took the giant demo Spark logo svg and put it into a variable so that it doesnt take up tons of lines of code in the "copy code" for our users.

Reply:
MINE TOO